.roave-backward-compatibility-check.xml

```diff
@@ -0,0 +1,50 @@
+          to be if they match the phpdoc. Parameter type changes will be reviewed manually by maintainers.
+        -->
+        <ignored-regex>#CHANGED: The parameter .+? changed from no type to#</ignored-regex>
+
+        <!-- these are `final public` methods and therefore safe to change even though the class is not final -->
+        <ignored-regex>#CHANGED: The return type of Behat\\Testwork\\Environment\\StaticEnvironment\#getSuite\(\) changed#</ignored-regex>
+        <ignored-regex>#CHANGED: The return type of Behat\\Testwork\\Environment\\Call\\EnvironmentCall\#getEnvironment\(\) changed#</ignored-regex>
+        <ignored-regex>#CHANGED: The return type of Behat\\Testwork\\Environment\\Call\\EnvironmentCall\#getCallee\(\) changed#</ignored-regex>
+        <ignored-regex>#CHANGED: The return type of Behat\\Testwork\\Environment\\Call\\EnvironmentCall\#getArguments\(\) changed#</ignored-regex>
+
+        <!-- Addition of strict return types is expected throughout the 4.0 release -->
+        <ignored-regex>#CHANGED: The return type of .+? changed from no type to#</ignored-regex>
+
+        <!--
+        When running in the 4.x branch, the BC checker installs symfony 8, therefore reports BC breaks in symfony
+        classes that we extend (since the 3.x release it compares to installed symfony 7)
+        -->
+        <ignored-regex>#REMOVED: Method Symfony\\Component\\Console\\Command\\Command.+? was removed#</ignored-regex>
+        <ignored-regex>#REMOVED: Method Symfony\\Component\\Console\\Application.+? was removed#</ignored-regex>
+
+        <!-- Individually list removed methods / constants (which were deprecated in 3.x) -->
+        <ignored-regex>#REMOVED: Method Behat\\Behat\\Output\\Node\\EventListener\\JUnit\\JUnitOutlineStoreListener\#getCurrentOutline\(\) was removed#</ignored-regex>
+        <ignored-regex>#REMOVED: Constant Behat\\Behat\\ApplicationFactory::VERSION#</ignored-regex>
+
+        <!--
+        Group changes to constructors (e.g. removing deprecated params) by class. The BC checker reports errors for each
+        parameter, but we don't need that level of granularity. It's enough to have a clear list of the classes that
+        have changed).
+        -->
+        <ignored-regex>#REMOVED: Method Behat\\Behat\\EventDispatcher\\Cli\\StopOnFailureController\#__construct\(\) was removed#</ignored-regex>
+        <ignored-regex>#CHANGED: The parameter .+? of Behat\\Behat\\Output\\Node\\Printer\\ListPrinter\#__construct\(\)#</ignored-regex>
+        <ignored-regex>#CHANGED: The parameter .+? of Behat\\Behat\\Output\\Node\\Printer\\JUnit\\JUnitScenarioPrinter\#__construct\(\)#</ignored-regex>
+        <ignored-regex>#CHANGED: (The parameter|Parameter) .+? of Behat\\Behat\\Definition\\Printer\\ConsoleDefinitionPrinter\#__construct\(\)#</ignored-regex>
+        <ignored-regex>#CHANGED: The parameter .+? of Behat\\Testwork\\Ordering\\Cli\\OrderController\#__construct\(\)#</ignored-regex>
```

Comment:
This merge brings in the backwards compatibility check in CI, which was of course failing as it is comparing to the latest tag (3.29.0) and the existing breaking changes already applied to the 4.x branch are not included in the baseline configuration.

c953cd3 updates the baseline to include these changes. Going forward we can update the baseline as we work on each 4.x PR.